doc/develop/toolchains/zephyr_sdk.rst

```diff
@@ -67,153 +67,173 @@ the recommended version for the corresponding Zephyr version.
 For the full list of compatible Zephyr and Zephyr SDK versions, refer to the
 `Zephyr SDK Version Compatibility Matrix`_.
 
-.. _toolchain_zephyr_sdk_install_linux:
+.. _toolchain_zephyr_sdk_install:
 
-Install Zephyr SDK on Linux
-***************************
+Zephyr SDK installation
+***********************
+
+.. toolchain_zephyr_sdk_install_start
+
+.. note:: You can change ``0.16.3`` to another version in the instructions below
+          if needed; the `Zephyr SDK Releases`_ page contains all available
+          SDK releases.
+
+.. note:: If you want to uninstall the SDK, remove the directory where you installed it.
+          If you relocate the SDK directory, you need to re-run the setup script.
```

Comment:
this is redundant as already mentioned at the end of the install instructions? I think it should be removed from one or the other(s), but I don't have a strong opinion as to which

Reply:
Fixed